### PR TITLE
Feature/btcp branding

### PR DESCRIPTION
**Pull request by Pega88**
Change appearance and branding from ZCL to BTCP.

NEEDS THOROUGH REVIEW TO CATCH TYPO'S OR COPY/PASTE ISSUES

### CODE REVIEW

File: src/resources/ubuntu-package/ZclassicWallet.desktop [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+Name=Bitcoin Private Wallet
+Comment=Bitcoin Private Desktop GUI Wallet (full node)
+GenericName=Bitcoin Private Wallet
+Keywords=BitcoinPrivate;cryptocurrency;BTCP;
+Exec=bitcoinpricat-desktop-gui-wallet
 Terminal=false
 X-MultipleArgs=false
 Type=Application
-Icon=/usr/lib/zclassic-gui-wallet/ZCL-icon.png
+Icon=/usr/lib/bitcoinpricate-gui-wallet/btcp-icon.png
```
Comment (rizkiwicaksono) [inline src/resources/ubuntu-package/ZclassicWallet.desktop]: @Pega88 typo, should be /usr/lib/bitcoinprivate-gui-wallet

**Comment by ch4ot1c** [comment]
Will do ZclassicSwingWallet-> ZclassicDesktopWallet tonight, and then we can changes here.

File: docs/TroubleshootingGuide.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-![Screenshot1](ZENChat_small.png "Chat Window") ![Screenshot1](ZclassicWalletMac_0.74.7_small.png "Wallet Window") 
+![Screenshot1](ZENChat_small.png "Chat Window") ![Screenshot1](BitcoinPrivateWalletMac_0.74.7_small.png "Wallet Window") 
 
 This document outlines possible solutions to several common problems that user might encounter while using the wallet.
 
 ### How to diagnose wallet problems
 
-When wallet problems occur, the information about the errors that caused the issue is usually found in the log files written by the wallet itself and also by `zcld`. The location of the wallet log files is:
+When wallet problems occur, the information about the errors that caused the issue is usually found in the log files written by the wallet itself and also by `btcp`. The location of the wallet log files is:
```
Comment (ch4ot1c) [inline docs/TroubleshootingGuide.md]: btcpd

File: docs/TroubleshootingGuide.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 ```
-Linux:    ~/.ZclassicSwingWallet/ZclassicGUIWallet_xxxx_xx_debug.log 
-Windows:  %LOCALAPPDATA%/ZclassicSwingWallet/ZclassicGUIWallet_xxxx_xx_debug.log
-Mac OS:   ~/Library/Application Support/ZclassicSwingWallet/ZclassicGUIWallet_xxxx_xx_debug.log
+Linux:    ~/.BitcoinPrivateSwingWallet/BitcoinPrivateGUIWallet_xxxx_xx_debug.log 
+Windows:  %LOCALAPPDATA%/BitcoinPrivateSwingWallet/BitcoinPrivateGUIWallet_xxxx_xx_debug.log
+Mac OS:   ~/Library/Application Support/BitcoinPrivateSwingWallet/BitcoinPrivateGUIWallet_xxxx_xx_debug.log
 ```
-`zcld` that is automatically started by the wallet, stores its logs at locations:
+`btcp` that is automatically started by the wallet, stores its logs at locations:
```
Comment (ch4ot1c) [inline docs/TroubleshootingGuide.md]: btcpd

File: docs/TroubleshootingGuide.md [hunk 4]
```diff
@@ -22,23 +22,23 @@ The log files are the first place too look for clues as to the nature of problem
 
 ### Common Problem 1 - wallet fails during start up
 
-This kind of problem may have multiple causes but the most frequent one in practice is that `zcld` fails to start properly. The latter is in turn most commonly caused by block-chain corruption. This could occur 
-in rare cases when for instance machines are stopped due to power disruptions, while `zcld` is writing data.
+This kind of problem may have multiple causes but the most frequent one in practice is that `btcp` fails to start properly. The latter is in turn most commonly caused by block-chain corruption. This could occur
```
Comment (ch4ot1c) [inline docs/TroubleshootingGuide.md]: btcp each next instance in this file

File: src/java/com/vaklinov/zcashui/AboutDialog.java [hunk 5]
```diff
@@ -76,14 +76,14 @@ public AboutDialog(JFrame parent)
 		licensePanel.setLayout(new BorderLayout(3, 3));
 		JTextArea licenseLabel = new JTextArea();
 		licenseLabel.setText(
-			"" + "Copyright (c) 2018 - The Zclassic Team\n\n"
+			"" + "Copyright (c) 2018 - The Bitcoin Private Team\n\n"
 		    + "Jon Layton <me@jonl.io>"
 		    + "\n"
-		    + "Donate ZCL: zcF3db2JwLNHa917NfbfFR2EJWXgowFmQ4bvouJEvGmPjLPcH7hyAhpSAFUhm7ANiBJfzMpJHMkp363r7M3GEC8g8oQXJ5n   "
+		    + "Donate BTCP: zcF3db2JwLNHa917NfbfFR2EJWXgowFmQ4bvouJEvGmPjLPcH7hyAhpSAFUhm7ANiBJfzMpJHMkp363r7M3GEC8g8oQXJ5n   "
```
Comment (ch4ot1c) [inline src/java/com/vaklinov/zcashui/AboutDialog.java]: remove these addrs

File: src/resources/ubuntu-package/ZclassicWallet.desktop (hunk 1 above)
Comment (Pega88) [reply src/resources/ubuntu-package/ZclassicWallet.desktop]: good catch both

File: docs/TroubleshootingGuide.md (hunk 2 above)
Comment (Pega88) [reply docs/TroubleshootingGuide.md]: got it

File: docs/TroubleshootingGuide.md (hunk 3 above)
Comment (Pega88) [reply docs/TroubleshootingGuide.md]: got it

File: docs/TroubleshootingGuide.md (hunk 4 above)
Comment (Pega88) [reply docs/TroubleshootingGuide.md]: got it

File: src/java/com/vaklinov/zcashui/AboutDialog.java (hunk 5 above)
Comment (Pega88) [reply src/java/com/vaklinov/zcashui/AboutDialog.java]: got it

**Comment by Pega88** [comment]
merged into develop rather than master

**Pull request closed without merging**